<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>addForm</class>
 <widget class="QWidget" name="addForm" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>174</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>232</width>
    <height>174</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>Add contact</string>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item row="0" column="0" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <property name="margin" >
       <number>9</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item row="3" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <spacer>
          <property name="orientation" >
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType" >
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" >
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QCheckBox" name="afaCheckBox" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="text" >
           <string>Ask for authorization</string>
          </property>
          <property name="shortcut" >
           <string/>
          </property>
          <property name="checked" >
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="2" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="groupLabel" >
          <property name="minimumSize" >
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="text" >
           <string>Group:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="groupLine" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
          <property name="editable" >
           <bool>true</bool>
          </property>
          <property name="autoCompletionCaseSensitivity" >
           <enum>Qt::CaseSensitive</enum>
          </property>
          <property name="frame" >
           <bool>true</bool>
          </property>
          <property name="modelColumn" >
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="loginLabel" >
          <property name="minimumSize" >
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="text" >
           <string>Login:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="loginLine" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="0" >
       <layout class="QHBoxLayout" >
        <property name="margin" >
         <number>0</number>
        </property>
        <property name="spacing" >
         <number>6</number>
        </property>
        <item>
         <widget class="QLabel" name="nameLabel" >
          <property name="minimumSize" >
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>60</width>
            <height>20</height>
           </size>
          </property>
          <property name="text" >
           <string>Name:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="nameLine" >
          <property name="minimumSize" >
           <size>
            <width>0</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize" >
           <size>
            <width>16777215</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <widget class="QPushButton" name="addButton" >
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton" >
       <property name="minimumSize" >
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>80</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <pixmapfunction></pixmapfunction>
 <tabstops>
  <tabstop>nameLine</tabstop>
  <tabstop>loginLine</tabstop>
  <tabstop>groupLine</tabstop>
  <tabstop>afaCheckBox</tabstop>
  <tabstop>addButton</tabstop>
  <tabstop>cancelButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
